apps/authz/kinds/v0alpha1/rolespec.cue

```diff
@@ -0,0 +1,23 @@
+package v0alpha1
+
+RoleSpec: {
+	#Permission: {
+		// RBAC action (e.g: "dashbaords:read")
+		action: string
+		// RBAC scope (e.g: "dashboards:uid:dash1")
+		scope: string
+	}
+	
+	name: string
+	displayName: string
```

Comment:
@radiohead have we established a convention for this? istr we were using `title`?

Reply:
Yeah, this should be `title`.